herd/BPFArch_herd.ml

```diff
@@ -54,6 +64,8 @@ struct
     | X -> "*"
     | SC -> "SC"
     | N -> ""
+    | A -> "ACQ"
+    | R -> "REL"
```

Comment:
Hi @puranjaymohan. This is not very important, but there is some inconsistencies in names here:  the cat name for the acquire (resp. release) effect set being `AQ` (resp. `RL`), while the annotation is pretty-printed as `ACQ` (resp. `REL`). Is it ok to keep names as they are?

Reply:
Hi @maranget 
I will fix it in the next update.